### DISCUSSION TITLE
Hello everyone!

### DISCUSSION THREAD

**Discussion by PastelBelem8** [OpenMined/courses#11] [Introduce Yourself]
Hey everyone!  

I'm Catarina Belém and I've been working as a Research Data Scientist at Feedzai for the last year and a half! I've enrolled in this course because I am very interested in learning more about these topics and because I see the potential of this line of research in the near future!

I'm really excited to learn and discuss these topics with you :)

**Comment by yemikifouly** [comment]
Hello Catarina! 🙂 It is great to have you here! Considering your interest on the topic,  you are definitely in the right place! Andrew and Emma explain things very clearly and it makes the topic a very interesting one to learn about.  🙂
